test/e2e_node/remote/utils.go

```diff
@@ -85,7 +85,8 @@ func configureFirewall(host string) error {
 	if err != nil {
 		return fmt.Errorf("failed to get iptables INPUT on %q: %v output: %q", host, err, output)
 	}
-	if strings.Contains(output, "Chain INPUT (policy DROP)") {
+	if strings.Contains(output, "Chain INPUT (policy DROP)") || !strings.Contains(output, "tcp") ||
```

Comment:
you can use 
```func configureFirewall(host tring) error {

output, err := SSH(host, ....
iptables -I INPUT 1 -w -p tcp -j ACCEPT
iptables -I INPUT 1 -w -p udp -j ACCEPT
iptables -I INPUT 1 -w -p icmp -j ACCEPT
iptables -I FORWARD 1 -w -p tcp -j ACCEPT
iptables -I FORWARD 1 -w -p udp -j ACCEPT
iptables -I FORWARD 1 -w -p icmp -j ACCEPT
)

return err
}
```

Reply:
this way you guarantee that the allowed rules are always at the top

Reply:
@aojea thanks. I just updated the PR with the suggested changes.